### ISSUE TITLE
CLIP-Sensor state not updated with automation

### ISSUE THREAD

**Issue by peterbartl** [dresden-elektronik/deconz-rest-plugin#1200]
I have read through the other issues related to CLIP-Sensors and my problem may or may not be what happens there. I am also fairly new to home automation in general.

Setup: 
Ikea Tradfri wireless remote (5 buttons)
Osram smart plug (connected to night stand lamp)
Xiaomi wired wall swich (switches mains powered ceiling light)

All three zigbee devices are connected via Phoscon. The center button is set to toggle the smart plug on and off. On its own, this works as expected. 
I have set an automation in home assistant to couple both lights. The idea is that I can switch on both lights from the wall swicht when I enter the room. This also works as expected. 
But then I want to use the wireless remote to switch off both lights with one button press (second automation). First press only changes the CLIP-Sensor state (from "on" to "off"), second press then actually switches off both lights.
So I assume that the CLIP-Sensor state is the issue here? How do I change this? I have tried it with an automation switching to an "on-scene" and with deconz.configure where I change the state of the smart plug. Both result in the behaviour described above. 

tldr:
Initial state: both lights are off, CLIP state "on"
press wall switch: both lights turn on, CLIP state "on"
press remote button once: both lights still on, CLIP state switches to "off"
press remote button again: both light turn off, CLIP state switches to "on"

Edit: I figured out a way to get the thing working. With an additional automation I can "manually" set the sensor state flag to "false" via deconz.configure.

{
  "data": {"flag": false },
  "field": "/sensors/10/state"
}

where 10 is the binary clip sensor in question.
For now I will use this workaround but I am still interested to learn what the correct version would be (or if this really is a bug).
